CLAUDE.md

```
@@ -273,6 +273,9 @@ This file contains high-level guidelines for Claude Code. For **more detailed, t
 - [Form Components](.cursor/rules/form-components.mdc) - Enhanced form components with authorization
 - [Testing Patterns](.cursor/rules/testing-patterns.mdc) - Testing strategies and examples
 
+### Implementation Guides
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)